backend/molgenis-emx2-io/src/main/java/org/molgenis/emx2/io/emx1/Emx1.java

```diff
@@ -187,19 +187,23 @@ private static Map<String, Emx1Entity> loadTables(TableStore store, SchemaMetada
 
   private static ColumnType getColumnType(String dataType) {
     switch (dataType) {
-      case "compound": // todo
+      case "compound": // unsupported, skipped.
```

Comment:
Why not let them go to default = string and remove these cases all together until you need them. Then you will see immediately that it is not implemented. Now it is implemented but it goes to a string.

Reply:
ah, good point. I can make them map to 'heading', so you would get a title to play with.